js/main.js

```diff
@@ -21,6 +25,15 @@ function getRandomInt(max) {
   return Math.floor(Math.random() * max);
 }
 
+function submitSearch(event) {
+  event.preventDefault();
+  var mobileSearch = $mobileSearchContent.value;
+  var desktopSearch = $desktopSearchContent.value;
+  if (mobileSearch || desktopSearch !== '') {
+    return mobileSearch || desktopSearch;
```

Comment:
Shouldn't you be sending a request to get data using the search term? Where is that code?